plugin-server/tests/worker/ingestion/action-matcher.test.ts

```diff
@@ -692,35 +678,23 @@ describe('ActionMatcher', () => {
-            const personFooBarabara = createTestPerson({ properties: { foo: 'barabara' } })
-            const personFooRabarbar = createTestPerson({ properties: { foo: 'rabarbar' } })
-            const personFooNumber = createTestPerson({ properties: { foo: 7 } })
-            const personNoNothing = createTestPerson()
-            const personFigNumber = createTestPerson({ properties: { fig: 999 } })
-            const personFooTrue = createTestPerson({ properties: { foo: true } })
-            const personFooNull = createTestPerson({ properties: { foo: null } })
-
-            expect(await actionMatcher.match(event, personFooBar)).toEqual([
+            expect(await actionMatcher.match({ ...event, person_properties: { foo: 'bar' } })).toEqual([
                 actionDefinitionOpExact,
                 actionDefinitionOpUndefined,
             ])
-            expect(await actionMatcher.match(event, personFooBarPolPot)).toEqual([
-                actionDefinitionOpExact,
-                actionDefinitionOpUndefined,
-            ])
-            expect(await actionMatcher.match(event, personFooBaR)).toEqual([])
-            expect(await actionMatcher.match(event, personFooBaz)).toEqual([])
-            expect(await actionMatcher.match(event, personFooBarabara)).toEqual([])
-            expect(await actionMatcher.match(event, personFooRabarbar)).toEqual([])
-            expect(await actionMatcher.match(event, personFooNumber)).toEqual([])
-            expect(await actionMatcher.match(event, personNoNothing)).toEqual([])
-            expect(await actionMatcher.match(event, personFigNumber)).toEqual([])
-            expect(await actionMatcher.match(event, personFooTrue)).toEqual([])
-            expect(await actionMatcher.match(event, personFooNull)).toEqual([])
+            // expect(await actionMatcher.match({...event, person_properties: {foo: 'bar', pol: 'pot' }})).toEqual([
+            //     actionDefinitionOpExact,
+            //     actionDefinitionOpUndefined,
+            // ])
+            // expect(await actionMatcher.match({...event, person_properties: {foo: 'baR'}})).toEqual([])
+            // expect(await actionMatcher.match({...event, person_properties: {foo: 'baz'}})).toEqual([])
+            // expect(await actionMatcher.match({...event, person_properties: {foo: 'barabara'}})).toEqual([])
+            // expect(await actionMatcher.match({...event, person_properties: {foo: 'rabarbar'}})).toEqual([])
+            // expect(await actionMatcher.match({...event, person_properties: {foo: 7}})).toEqual([])
+            // expect(await actionMatcher.match({...event, person_properties: {}})).toEqual([])
+            // expect(await actionMatcher.match({...event, person_properties: {something_else: 999}})).toEqual([])
+            // expect(await actionMatcher.match({...event, person_properties: {foo: true}})).toEqual([])
+            // expect(await actionMatcher.match({...event, person_properties: {foo: null}})).toEqual([])
```

Comment:
Why's this commented out? This shouldn't be committed

Reply:
ah ... I was testing something and forgot ... good spot